src/app/components/buttons/button.module.css

```diff
@@ -32,13 +32,13 @@
   align-items: center;
   justify-content: center;
 
-  width: 312px;
+  /* width: 312px; */
```

Comment:
Which issue is being addressed here? Remove the comments, if you have a note that you think is important that it  stays inside, put it there. 

In the future, if you feel you need to temporarily comment out a few things and is related to an issue that we ought to solve - instead of commenting, we will create a card in the Trello board so that it stays documented and doesn't get lost. You can discuss the issue with me if needed.